app/views/layouts/application.html.erb

```diff
@@ -13,20 +13,20 @@
 
 <% content_for :body_end do %>
   <script>
-  var loadedCalendars = [];
+  var loadedModules = [];
   $(document).on('turbolinks:load', function() {
-    var calendars = $('[data-calendar]');
+    var modules = $('[data-module]');
 
-    if (loadedCalendars.length) {
+    if (loadedModules.length) {
```

Comment:
Yep! Maybe store if the module has been loaded on the corresponding div like `data-loaded-modules='whatevs'`.